scenarioo-server/src/main/java/org/scenarioo/rest/builds/BuildsImporterResource.java

```diff
@@ -98,6 +95,35 @@ public void reimportBuild(@PathParam("branchName") final String branchName,
 		ScenarioDocuBuildsManager.INSTANCE.reimportBuild(buildIdentifier);
 	}
 
+	@GET
+	@Path("importBuild/{branchName}/{buildName}/{comparisonBranchName}/{comparisonBuildName}/{comparisonName}")
+	@Produces({"application/json"})
+	public Response importBuild(
```

Comment:
@til77 yes, sure, there is already such an "id" and this is implicitly given by "branch name" + "build name" + the configured "comparison name" - and there is a also a REST service to get the result once it is calculated. Would that be enough?